packages/@aws-cdk/aws-appmesh/test/test.route.ts

```diff
@@ -574,6 +574,51 @@ export = {
 
       test.done();
     },
+
+    'with shared service mesh': {
+      'Mesh Owner is the AWS account ID of the account that shared the mesh with your account'(test: Test) {
+        // GIVEN
+        const app = new cdk.App();
+        const accountA = { account: '1234567899', region: 'us-west-2' };
+        const accountB = { account: '9987654321', region: 'us-west-2' };
```

Comment:
1. This is called an `env` in CDK (the name `account` doesn't make sense, since this also contains the `region`).
2. Can you call this `routeEnv` instead?